tasks/lib/phantomas.js

```diff
@@ -319,19 +326,15 @@ Phantomas.prototype.executePhantomas = function() {
         options[ 'film-strip-dir' ] = this.imagePath + this.timestamp;
       }
 
-      runs.push(
-        phantomas(
-          this.options.url,
-          options
-        )
-      );
+      runs.push( callPhantomas( this.options.url, options ) );
     }
 
-    Promise.settle( runs )
-          .then( resolve )
-          .catch( function( e ) {
-            console.log( e );
-          } );
+    Promise.reduce( runs, function( total, run, index ) {
```

Comment:
Hey hey,

maybe I'm wrong, but an `each` would do the same trick or?
There is actually no reduction.

https://github.com/petkaantonov/bluebird/blob/master/API.md#eachfunction-iterator---promise